JSDemos/Demos/TreeList/UsingHeaderFilter/description.md

```diff
@@ -1 +1,7 @@
-This demo shows how to&nbsp;use the header filter for filtering data. The header filter contains a&nbsp;list of&nbsp;unique column values that you can select. To&nbsp;open this list, click the filter icon in&nbsp;a&nbsp;column header. Once you select a&nbsp;value from this list and click&nbsp;OK, the filter is&nbsp;applied, and the TreeList displays the full hierarchy down to&nbsp;matching rows, but not further.
\ No newline at end of file
+A header filter allows users to select column values from a predefined set and use the selection to filter rows. To enable this behavior, set **headerFilter**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/#visible) to **true**. This feature can be combined with the [filter row](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/UsingFilterRow).
+
+Once the header filter is enabled, a funnel icon is added to each column's header. To apply a filter, click this icon, choose value(s) and hit OK. You can use the **columns[]**.[headerFilter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/) property to customize an individual filter. You can also disable it: for this, set **columns[]**.[allowHeaderFiltering](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowHeaderFiltering) to **false**.
```

Comment:
1. Let's get rid of passive and clarify that some actions are performed by users:.
1. > to customize an individual filter

I think that it's unclear which filter we mean - a filter from a drop-down list or a full header filter. Maybe we can remove this ambiguity?
1. >for this

Vladimir once found this phrase in my text and told me that this sounds like a bad translation from Russian. Let's get rid of it too.

```suggestion
When you enable the header filter, a funnel icon appears in each column's header. Users can click this icon to open the drop-down list and choose a value to filter by. You can use the **columns[]**.[headerFilter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/) property to configure individual header filters. Set **columns[]**.[allowHeaderFiltering](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowHeaderFiltering) to **false** to disable a header filter.
```